platform/engineering/design-docs/2020-04-09-separate-content-build.md

```diff
@@ -208,18 +207,10 @@ which corresponds to the URLs on VA.gov. For example, `/education/index.html` is
 the file that's served at www.va.gov/education/.
 
 The output of the application build **currently** live in the `/generated/`
-directory. To support the transition, the **new** application build will put its
-assets in the `/applications/` directory.
+directory. By using different buckets for the application build output and content build output, the `/generated` directory can be removed.
```

Comment:
How will requests to the content be routed to the content bucket? How will requests for application assets be routed to the application bucket?